pydantic/main.py

```diff
@@ -141,6 +143,16 @@ def __new__(mcs, name, bases, namespace):
         }
         return super().__new__(mcs, name, bases, new_namespace)
 
+    @staticmethod
+    def _validate_field_name(bases: List[Type['BaseModel']], field_name: str) -> None:
```

Comment:
I don't think this needs to be a static method on the class. Just a simple method in utils would be better.